spec/rubocop/cop/rspec/scattered_setup_spec.rb

```diff
@@ -7,22 +7,22 @@
     expect_offense(<<-RUBY)
       describe Foo do
         before { bar }
-        ^^^^^^^^^^^^^^ Do not define multiple hooks in the same example group.
+        ^^^^^^^^^^^^^^ Do not define multiple `before` hooks in the same example group (defined on lines 2, 3).
```

Comment:
Would it be possible to only list the other lines, not self? (See https://github.com/rubocop-hq/rubocop-rspec/pull/860/files)

Reply:
Changes pushed.